test/e2e/testing-manifests/storage-csi/gce-pd/node_ds.yaml

```diff
@@ -18,11 +18,11 @@ spec:
           args:
             - "--v=5"
             - "--csi-address=/var/lib/kubelet/plugins/pd.csi.storage.gke.io/csi.sock"
-            - "--kubelet-registration-path=/var/lib/kubelet/plugins/pd.csi.storage.gke.io/csi.sock"
+            - "--kubelet-registration-path=/registration/pd.csi.storage.gke.io/csi.sock"
           lifecycle:
             preStop:
               exec:
-                command: ["/bin/sh", "-c", "rm -rf /registration/pd.csi.storage.gke.io /registration/pd.csi.storage.gke.io-reg.sock"]
+                command: ["/bin/sh", "-c", "rm -rf /registration/pd.csi.storage.gke.io /var/lib/kubelet/plugins/pd.csi.storage.gke.io/pd.csi.storage.gke.io-reg.sock"]
```

Comment:
Undo this change. these paths are internal to the driver container, not on the host.